commands/bosh_environment.go

```diff
@@ -91,6 +92,10 @@ func (be BoshEnvironment) Execute(args []string) error {
 	}
 
 	variables := make(map[string]string)
+	unsetVariables := []string{
+		"BOSH_ALL_PROXY",
```

Comment:
See comment in Conversation. This seems like a potential problem if you are only unsetting credhub or bosh vars.

Reply:
I'll respond in the conversation about them.